garage/tf/core/mlp.py

```diff
@@ -1,12 +1,11 @@
 """
-Multilayer Perceptrons (MLP) with tensorflow as the only dependency.
+MLP with tensorflow as the only dependency.
 
-The module contains MLP which serves as the base of all networks.
-It aims to replace existing implementation of MLP class
+The module contains MLP implementation.
+It aims to replace existing implementation of network class
```

Comment:
Editorial comments about the development process aren't usually necessary in documentation.